config.default.yaml

```diff
@@ -9,31 +9,49 @@ logging:
   level: INFO
   format: "%(levelname)s:%(name)s:%(message)s"
 
+results_dir: results/
+summary_dir: results/
+costs_dir: data/ # TODO change to the equivalent of technology data
+
+foresight: overnight
+
+
 countries: ["NG", "BJ"]
 # Can be replaced by country ["NG", "BJ"], continent ["Africa"] or user-specific region, see more at https://pypsa-earth.readthedocs.io/en/latest/configuration.html#top-level-configuration
 
 enable:
-  retrieve_databundle: true  #  Recommended 'true', for the first run. Otherwise data might be missing.
-  retrieve_cost_data: true  # true: retrieves cost data from technology data and saves in resources/costs.csv, false: uses cost data in data/costs.csv
-  download_osm_data: true  # If 'true', OpenStreetMap data will be downloaded for the above given countries
-  build_natura_raster: false  # If True, then an exclusion raster will be build
+  retrieve_databundle: true #  Recommended 'true', for the first run. Otherwise data might be missing.
+  retrieve_cost_data: true # true: retrieves cost data from technology data and saves in resources/costs.csv, false: uses cost data in data/costs.csv
+  download_osm_data: true # If 'true', OpenStreetMap data will be downloaded for the above given countries
+  build_natura_raster: false # If True, then an exclusion raster will be build
   build_cutout: false
   # If "build_cutout" : true, then environmental data is extracted according to `snapshots` date range and `countries`
   # requires cds API key https://cds.climate.copernicus.eu/api-how-to
   # More information https://atlite.readthedocs.io/en/latest/introduction.html#datasets
 
-custom_rules: []  # Default empty [] or link to custom rule file e.g. ["my_folder/my_rules.smk"] that add rules to Snakefile
+# CI relevant
+retrieve_databundle: # required to be "false" for nice CI test output
```

Comment:
We have this twice currently, one for the Earth and one for the Earth-Sec databundle. How about we name it accordingly, e.g. `retrieve_databundle` and `retrieve_databundle_sector` and move both under the section `enable`?

Reply:
good catch!